cmd/buildah/main.go

```diff
@@ -78,6 +78,8 @@ func init() {
 	}
 
 	cobra.OnInitialize(initConfig)
+	// Disable the implicit `completion` command in cobra.
+	rootCmd.CompletionOptions.DisableDefaultCmd = true
```

Comment:
What was this for?  It wasn't mentioned in the commit log or this PR description.

Reply:
I have no idea.  It's too long ago.

I assume something failed or a behavior changed.  Cobra was bumped in this PR as well.

Reply:
Best to remove it and see what breaks.